retool_test.go

```diff
@@ -98,6 +98,24 @@ func TestRetool(t *testing.T) {
 			}
 		})
 
+		t.Run("build_with_gobin_set", func(t *testing.T) {
+			// Set GOBIN to a directory not controlled by retool. It should still
```

Comment:
I find the comment a bit confusing and had to reread it. Maybe "it" should be "retool" to make it clearer?